freeipa.spec.in

```diff
@@ -1241,10 +1241,12 @@ if [ $1 -gt 1 ] ; then
             cp /etc/ipa/ca.crt /var/lib/ipa-client/pki/kdc-ca-bundle.pem
             cp /etc/ipa/ca.crt /var/lib/ipa-client/pki/ca-bundle.pem
         fi
+
+        %{python} -c 'from ipaclient.install.client import configure_krb5_snippet; configure_krb5_snippet()' >>/var/log/ipaupgrade.log 2>&1
     fi
```

Comment:
We might want/need an ipa-client-upgrade script to do this instead, if at least for other distros (perhaps outside the scope of this PR).

Reply:
That was my thought as well.  We'd probably want to include the cert stuff above it if we ever make such a tool.